java/arcs/core/data/expression/PaxelParser.kt

```diff
@@ -80,16 +80,18 @@ object PaxelParser : Grammar<Expression<Any>>() {
   private val colon by -regex("($WS*:$WS*)")
 
   private val units by
-    optional(whitespace +
-      (regex("(millisecond)s?").map { Unit.Millisecond } /
-        regex("(second)s?").map { Unit.Second } /
-        regex("(minute)s?").map { Unit.Minute } /
-        regex("(hour)s?").map { Unit.Hour } /
-        regex("(day)s?").map { Unit.Day }
+  optional(
+    whitespace +
+      (
+        regex("(millisecond)s?").map { Unit.Millisecond } /
+          regex("(second)s?").map { Unit.Second } /
+          regex("(minute)s?").map { Unit.Minute } /
+          regex("(hour)s?").map { Unit.Hour } /
+          regex("(day)s?").map { Unit.Day }
         )
```

Comment:
really regrettable that it can't figure this out, but the closing ')' doesn't line up.